tutorials/applications/recipes/loading_text_files.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>Same with comment on the Pandas implementation above. We might be able to condense the Spark/Dask/Pandas implementations here. </p><p> </p><p>Just a tip for running with Dask also. There are these weird errors but if you re-run the cell twice, you can get rid of them which is just nicer for docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>